### ISSUE TITLE
Bug in column aliasing for PostgreSQL

### ISSUE THREAD

**Issue by zachsa** [dbeaver/dbeaver#8900]
Using PostGIS (same server for DBeaver vs PgAdmin), and using DBeaber 7.1.0

Using PgAdmin 4, I can run this query:

```sql
select
id,
"name",
ST_AsGeoJSON(st_transform(xyz, 4326)) xyz
from public.sites
```

And I get values back for the `xyz` column (note that I'm aliasing the transformed column with the same name as the original column).

However this same query in DBeaver returns only `[null]` for the `xyz` column in the result. Instead if I adjust the query to be:

```sql
select
id,
"name",
ST_AsGeoJSON(st_transform(xyz, 4326)) xyz_
from public.sites
```

Then I get results. There must be a bug in column aliasing in the driver used by DBeaver, or a bug in DBeaver.

**Comment by uslss** [comment]
thanks for the bug report

**Comment by serge-rider** [comment]
I think it was fixed in #8304

**Comment by uslss** [comment]
verified
